.github/workflows/pull-request-update.yml

```diff
@@ -0,0 +1,56 @@
+name: Pipeline
+on: pull_request
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@master
+      - name: Build dev image and save it locally
+        uses: docker/build-push-action@v2
+        env:
+          BUILDKIT_PROGRESS: plain
+        with:
+          context: ./
+          file: ./docker/pipeline/Dockerfile
+          builder: ${{ steps.buildx.outputs.name }}
+          target: builder
+          tags: test/img
+          outputs: type=local,dest=./img
+          build-args: RUN_TESTS=${{ true }}
+          cache-from: type=registry,ref=${{ secrets.DOCKER_USERNAME }}/aam-digital:cache
+      - name: Publish code coverage
+        uses: paambaati/codeclimate-action@v2.7.5
+        env:
+          CC_TEST_REPORTER_ID: ${{ secrets.CODE_CLIMATE_ID }}
+        with:
+          coverageLocations: ${{github.workspace}}/img/app/coverage/lcov.info:lcov
+  deploy-prod-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Docker Buildx
```

Comment:
there is some duplication here with the setup for building on master. Is there any feasible way to let them use the same config?

Reply:
I dont think so and you can not refer to a job in a different Workflow